lib/pages/contacts/contacts_data.dart

```diff
@@ -24,4 +24,4 @@ abstract class _ContactsData with Store {
     loaded = true;
   }
   
-}
\ No newline at end of file
+}
```

Comment:
Restore newline at end of file

```suggestion
}

```